docs/CONTRIBUTING.md

```diff
@@ -139,9 +139,8 @@ setup properly.
 
 ### asdf-vm Setup
 
-We use [asdf-vm](https://asdf-vm.com) to manage language versions for Firezone.
-Install the language runtimes defined in the [.tool-versions](../.tool-versions)
-file by running `asdf install` from the project root.
+If you have `asdf` installed, you can install the required language runtimes by
```

Comment:
I adapted the language here because it is by no means necessary to use `asdf` to contribute to Firezone. I don't use it and in fact, I think it is odd to mandate such tools.

Reply:
This won't work because the runtime won't be activated until you are _in_ the directory or one of its children.